src/TestProblem/Hydro/ClusterMerger/Init_TestProb_ClusterMerger.cpp

```diff
@@ -179,6 +256,15 @@ void SetParameter()
    ReadPara->Add( "Merger_File_Par2",        Merger_File_Par2,       NoDef_str,        Useless_str,   Useless_str    );
    ReadPara->Add( "Merger_File_Prof3",       Merger_File_Prof3,      NoDef_str,        Useless_str,   Useless_str    );
    ReadPara->Add( "Merger_File_Par3",        Merger_File_Par3,       NoDef_str,        Useless_str,   Useless_str    );
+   ReadPara->Add( "Unit_R1",                &Unit_R1,                1.0,              NoMin_double,  NoMax_double   );
```

Comment:
See my comment on `Input__TestProb`. These are superfluous since in the latest version of `cluster_generator` we have a function to generate ICs just for GAMER and it always outputs the profiles in CGS units.